Sources/Components/Control/ReviewButtonControl/ReviewButtonControl.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Copyright © 2019 FINN AS. All rights reserved.
+//
+
+import UIKit
+
+public protocol ReviewButtonControlDelegate: AnyObject {
+    func reviewButtonControl(_ reviewButtonControl: ReviewButtonControl, giveReviewWasTapped startReview: Bool)
+}
+
+public final class ReviewButtonControl: UIControl {
+    // MARK: - Public
+
+    public weak var delegate: ReviewButtonControlDelegate?
+    public var text: String = "" {
+        didSet {
+            titleLabel.text = text
+        }
+    }
+    public static let cornerRadius: CGFloat = 8
+
+    // MARK: - Private
+
+    private lazy var imageView: UIImageView = {
+        let image = UIImage(named: .rate).withRenderingMode(.alwaysTemplate)
+        let imageView = UIImageView(image: image)
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        imageView.backgroundColor = .clear
+        imageView.tintColor = .milk
+        imageView.clipsToBounds = true
+        imageView.contentMode = .scaleAspectFit
+        return imageView
+    }()
+
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .bodyStrong)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .milk
+        label.textAlignment = .center
+        return label
+    }()
+
+    // MARK: - Init
+
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+    }
+
+    public required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    // MARK: - Override
+
+    public override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        super.touchesBegan(touches, with: event)
+        UIView.animate(withDuration: 0.0, animations: {
```

Comment:
Setting the duration to 0.0 makes this a non-animating-animation™. Maybe you should just set the values and not bother with the `.animate` stuff(?)
(same thing in the `touchesEnded`, I guess). :-)